inc/commonitilobject.class.php

```diff
@@ -6657,7 +6657,71 @@ function viewEditSubitem" . $this->fields['id'] . "$rand(e, itemtype, items_id,
 
       echo "</ul>"; // timeline_choices
       echo "<div class='clear'>&nbsp;</div>";
-
+      //total_actiontime stat
+      if (Session::getCurrentInterface() != 'helpdesk') {
+         echo "<div class='timeline_stats'>";
+         echo "<h3>";
+         $total_actiontime = 0;
+         foreach ($DB->request("glpi_tickettasks",
```

Comment:
Do not call `request` inside of a loop; store the restul in a variable before and use that variable in the loop (for all loops).

Reply:
Also, do **not** use raw SQL.

Reply:
Better ?

Reply:
:+1: